exchange/utils.go

```diff
@@ -686,7 +686,30 @@ func getExtTargetData(requestExt *openrtb_ext.ExtRequest, cacheInstructions *ext
 	return targData
 }
 
-func getDebugInfo(bidRequest *openrtb2.BidRequest, requestExt *openrtb_ext.ExtRequest) bool {
+// getDebugInfo returns the boolean flags that allow for debug information in bidResponse.Ext, the SeatBid.httpcalls slice, and
+// also sets the debugLog information
+func getDebugInfo(bidRequest *openrtb2.BidRequest, requestExt *openrtb_ext.ExtRequest, accountDebugFlag bool, debugLog *DebugLog) (bool, bool, *DebugLog) {
```

Comment:
Despite `getDebugInfo `  function has a test coverage because of `HoldAuction` tests, can you please add separate unit test(s) for it?

Reply:
Good catch. Added `TestGetDebugInfo` and refactored `getDebugInfo` a bit (hope it's a bit more readable)